extensions/amp-story/1.0/amp-story.js

```diff
@@ -1699,6 +1699,13 @@ export class AmpStory extends AMP.BaseElement {
     const vState = this.getAmpDoc().getVisibilityState();
     const isPreview = vState === VisibilityState_Enum.PREVIEW;
     const isVisible = vState === VisibilityState_Enum.VISIBLE;
+
+    if (isPreview) {
+      this.storeService_.dispatch(Action.TOGGLE_SYSTEM_UI_IS_VISIBLE, false);
+    } else if (isVisible) {
```

Comment:
Sorry about this, just a personal and unjustified preference (I usually struggle to follow `else` statements)

```suggestion
    }
    if (isVisible) {
```